### ISSUE TITLE
YAML to JSON conversion unnecessary and broken

### ISSUE THREAD

**Issue by doismellburning** [kickstarter/cfn-flow#25]
CloudFormation now supports YAML templates natively, rendering `cfn-flow`'s "converts from YAML to JSON" obsolete

In addition, CloudFormation Intrinsic Functions have a YAML "short form" (e.g. `!GetAtt foo.bar`) that `cfn-flow` naïvely transforms to JSON, turning a valid CF YAML template into invalid CF JSON

(If you confine yourself to the "long form" of the functions, it's fine)

**Comment by ktheory** [comment]
@doismellburning Yup! Totally correct. I'd welcome a PR to removes the conversion to JSON.
